.tx/config

```diff
@@ -1,7 +1,7 @@
 [main]
 host = https://www.transifex.com
 
-[qgep-project.qgep_ents]
+[o:qgep-project:p:qgep_ents]
```

Comment:
@why-not-try-calmer can you add a short explanation what's being done here / why? Is that a new notation?

Reply:
Correct. 

- API v.2 (deprecated): `[<project>.<resource>]`
 - API v.3 (latest): `[o:<organization>:p:<project>:r:<resource>]`